src/transaction/celo.ts

```diff
@@ -158,11 +211,12 @@ export const prepareCeloTransferErc721SignedTransaction = async (testnet: boolea
         feeCurrency,
         nonce,
         signatureId,
+        value
     } = body;
 
     const p = new CeloProvider(provider || `${TATUM_API_URL}/v3/celo/web3/${process.env.TATUM_API_KEY}`);
     const network = await p.ready;
-
+    const client=await getClient(provider?provider:`${TATUM_API_URL}/v3/celo/web3/${process.env.TATUM_API_KEY}`)
```

Comment:
dont need to instantiate it here. utils package can be imported as import {toWei} from 'web3-utils'